### DISCUSSION TITLE
MRI alignment

### DISCUSSION THREAD

**Discussion by FrancescoLomi** [simnibs/simnibs#391] [7.Creating head models]
Dear all,
Do you know if, as a preliminary step, is it necessary to adjust the position of the mri image before entering the freesurfer and simnibs pipeline? I mean, some of my subjects were not correctly positioned in the scanner, I wonder if this could create some issues in the following steps.
Thank you!

**Comment by oulap** [comment]
In charm you can tune the affine registration between the atlas and the image, see here:

https://simnibs.github.io/simnibs/build/html/tutorial/advanced/fix_affine_registration.html#fix-affine-registration-tutorial

So you don't need to transform the image itself before processing it with charm. You can just tweak the initial affine fit parameters to improve the registration. I think FreeSurfer is pretty robust to these things as well, so I would just give it a shot.
